apps/reaper/lib/reaper/auth_retriever.ex

```diff
@@ -4,6 +4,26 @@ defmodule Reaper.AuthRetriever do
   """
   alias Reaper.Collections.Extractions
   alias Reaper.Cache.AuthCache
+  alias Reaper.UrlBuilder
+
+  def authorize(dataset_id, url, body, encode_method, headers, cache_ttl) do
+    cache_ttl = cache_ttl || 10_000
```

Comment:
the function below this defaults the cache_ttl in the function signature, we should probably be consistent

Reply:
Unfortunately no.  This guy is called by dataset definitions using EEX and sometimes they dont pass it at all.  Since we put it as a field and want to pass the field through (making this module not aware of "steps") we have to handle it different in the new world.  In the new world this function disappears though as long as we migrate datasets which use it.